docs/toc.md

```diff
@@ -292,10 +292,6 @@
 ### [🔧 Reflection & code generation](csharp/reflection.md)
 ### [Documenting your code](csharp/codedoc.md) 
 ## [🔧 Using the .NET Compiler Platform](csharp/roslyn/index.md)
-## [Language Reference](csharp/language-reference/index.md)
```

Comment:
yes, the link already exists so this one added by @cartermp should just be removed.